feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/handling/types/DebugNotificationHandler.kt

```diff
@@ -0,0 +1,57 @@
+package io.novafoundation.nova.feature_push_notifications.data.presentation.handling.types
+
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.google.firebase.messaging.RemoteMessage
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.feature_push_notifications.BuildConfig
+import io.novafoundation.nova.feature_push_notifications.R
+import io.novafoundation.nova.feature_push_notifications.data.presentation.handling.NotificationHandler
+import io.novafoundation.nova.feature_push_notifications.data.presentation.handling.buildWithDefaults
+
+private const val DEBUG_NOTIFICATION_ID = -1
+
+/**
+ * A [NotificationHandler] that is used as a fallback if previous handlers didn't handle the notification
+ */
+class DebugNotificationHandler(
+    private val context: Context,
+    private val notificationManager: NotificationManagerCompat,
+    private val resourceManager: ResourceManager
+) : NotificationHandler {
+
```

Comment:
Added this class for debugging to make sure that push is arrived in case when any other handler didn't handle it